internal/report/report_test.go

```diff
@@ -330,7 +330,7 @@ func TestComputeTotal(t *testing.T) {
 		{
 			Location: []*profile.Location{testL[2], testL[1], testL[0]},
 			Value:    []int64{-10, 3},
-			Label:    map[string][]string{"pprof::base": {"true"}},
+			Label:    map[string][]string{profile.DiffBaseLabelKey: {profile.DiffBaseLabelValue}},
```

Comment:
I would not make this change. https://github.com/google/pprof/pull/384#discussion_r192537247 applied to all tests.

Reply:
Sorry -- missed this test.
Should all be fixed now.